setup-r-dependencies/action.yaml

```diff
@@ -56,16 +56,15 @@ runs:
               pak::pkg_system_requirements(dep, execute = TRUE)
             }
           }
-          pak::pkg_install(c(local_deps, needs_only_deps, extra_deps, "sessioninfo"))
+          pak::pkg_install(c(local_deps, needs_only_deps, extra_deps, "r-lib/sessioninfo@a68df12"))
           cat("::endgroup::\n")
         shell: Rscript {0}
 
       - name: Session info
         run: |
           cat("::group::Session info\n")
           options(width = 100)
-          pkgs <- installed.packages()[, "Package"]
-          sessioninfo::session_info(pkgs, include_base = TRUE)
+          sessioninfo::session_info("!installed", include_base = TRUE)
```

Comment:
What does `!installed` mean here? I would take that as 'not installed', which seems to be the opposite of our intent?

Reply:
It is showing the installed packages.

Reply:
That is unfortunate syntax if that is the case.

Reply:
Yeah, I agree. I'll change it to just `"installed"` in the next sessioninfo release, which is coming tomorrow, because CRAN is making me update it. It is unlikely that we'll have a package called "installed".